util/runtests.sh

```diff
@@ -149,7 +157,8 @@ SIMPATH="$ODEMIS_DIR/install/linux/usr/share/odemis/sim/"
 
 ODMPATH="$ODEMIS_DIR/../mic-odm-yaml/" # extra microscope files
 if [ -d "$ODMPATH" ]; then
-    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" "$ODMPATH"/*/ >> "$TESTLOG" 2>&1
+    "$ODEMIS_DIR/util/run_intg_tests.py" --log-path "$INTEGLOGDIR" --interpreter "/usr/bin/python" "$ODMPATH"/*/ >> "$TESTLOG-python2" 2>&1
```

Comment:
Let's write explicitly python2, to be 100% sure the right Python starts.

Reply:
done